afl-fuzz [options] -- /path/to/ht @@

--- a/main.cc
+++ b/main.cc
@@ -264,6 +264,15 @@
 
 int main(int argc, char *argv[])
 {
+	int pipefd[2];
+	if (pipe(pipefd))
+	  abort();
+	if (dprintf(pipefd[1], "\x1b[21~") != 5)
+	  abort();
+	if (dup2(pipefd[0], 0))
+	  abort();
+	close(pipefd[0]);
+	close(pipefd[1]);
 #if defined(WIN32) || defined(__WIN32__)
 	HMODULE h = GetModuleHandle(NULL);
 	GetModuleFileName(h, appname, sizeof appname-1);
