--- a/src/framing.c
+++ b/src/framing.c
@@ -23,6 +23,7 @@
 #include <stdlib.h>
 #include <limits.h>
 #include <string.h>
+#include <stdio.h>
 #include <ogg/ogg.h>
 
 /* A complete description of Ogg framing exists in docs/framing.html */
@@ -718,6 +719,11 @@
 
     /* Compare */
     if(memcmp(chksum,page+22,4)){
+      fprintf(stderr, "NOTE: checksum mismatch: %02hhx%02hhx%02hhx%02hhx -> %02hhx%02hhx%02hhx%02hhx\n",
+         chksum[0], chksum[1], chksum[2], chksum[3],
+         page[22], page[23], page[24], page[25]
+      );
+#if 0
       /* D'oh.  Mismatch! Corrupt page (or miscapture and not a page
          at all) */
       /* replace the computed checksum with the one actually read in */
@@ -725,6 +731,7 @@
 
       /* Bad checksum. Lose sync */
       goto sync_fail;
+#endif
     }
   }
 
