afl-fuzz [options] -- /path/to/mutt -R -f @@ -e 'set mailcap_path='

--- a/main.c
+++ b/main.c
@@ -596,6 +598,16 @@
   extern int optind;
   int double_dash = argc, nargc = 1;
 
+  int pipefd[2];
+  if (pipe(pipefd))
+    abort();
+  if (dprintf(pipefd[1], "\7\nhqq") != 5)
+    abort();
+  if (dup2(pipefd[0], 0))
+    abort();
+  close(pipefd[0]);
+  close(pipefd[1]);
+
   /* sanity check against stupid administrators */
   
   if(getegid() != getgid())
@@ -795,12 +807,14 @@
       exit (0);
   }
 
+#if 0
   /* Check for a batch send. */
   if (!isatty (0) || queries || alias_queries || dump_variables)
   {
     set_option (OPTNOCURSES);
     sendflags = SENDBATCH;
   }
+#endif
 
   /* This must come before mutt_init() because curses needs to be started
      before calling the init_pair() function to set the color scheme.  */
--- a/mx.c
+++ b/mx.c
@@ -21,6 +21,8 @@
 # include "config.h"
 #endif
 
+#undef USE_DOTLOCK
+
 #include "mutt.h"
 #include "mx.h"
 #include "rfc2047.h"
